.github/workflows/build-lint-test.yml

```diff
@@ -86,13 +111,20 @@ jobs:
         uses: actions/cache@v3
         with:
           path: |
-            packages/snaps-simulator/dist/vendor
+            packages/snaps-simulator/vendor
           key: webpack-vendor-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
       - name: Build Webpack vendor
         if: steps.cache-webpack-vendor.outputs.cache-hit != 'true'
         run: yarn workspace @metamask/snaps-simulator run build:vendor
-      - name: Build Webpack
-        run: yarn workspace @metamask/snaps-simulator run build
+      - name: Build
+        run: yarn workspace @metamask/snaps-simulator run build:webpack
+      - name: Cache "@metamask/snaps-simulator" build
```

Comment:
Should we remove this caching since we wont use it yet?

Reply:
Not sure if it's worth the effort. Caching takes very little time anyway, and we will need it for #1488.